assets/xml/objects/object_ahg.xml

```diff
@@ -1,53 +1,61 @@
-        <Limb Name="object_ahg_Standardlimb_005950" Type="Standard" EnumName="OBJECT_AHG_LIMB_0F" Offset="0x5950" />
-        <Skeleton Name="object_ahg_Skel_005998" Type="Flex" LimbType="Standard" LimbNone="OBJECT_AHG_LIMB_NONE" LimbMax="OBJECT_AHG_LIMB_MAX" EnumName="ObjectAhgLimb" Offset="0x5998" />
+        <Texture Name="gAhgTLUT" OutName="tlut" Format="rgba16" Width="16" Height="16" Offset="0x0" />
+        <Texture Name="gAhgHairLineTex" OutName="hair_line" Format="ci8" Width="16" Height="16" Offset="0x200" />
+        <Texture Name="gAhgHairTex" OutName="hair" Format="ci8" Width="16" Height="16" Offset="0x300" />
+        <Texture Name="gAhgEarTex" OutName="ear" Format="ci8" Width="8" Height="8" Offset="0x400" />
+        <Texture Name="gAhgSkinGradientTex" OutName="skin" Format="ci8" Width="8" Height="8" Offset="0x440"/>
+        <Texture Name="gAhgEyeOpenTex" OutName="eye_open" Format="ci8" Width="16" Height="16" Offset="0x480" TlutOffset="0x0"/>
+        <Texture Name="gAhgEyeHalfTex" OutName="eye_half" Format="ci8" Width="16" Height="16" Offset="0x580" TlutOffset="0x0"/>
+        <Texture Name="gAhgEyeClosedTex" OutName="eye_closed" Format="ci8" Width="16" Height="16" Offset="0x680" TlutOffset="0x0"/>
+        <Texture Name="gAhgFingersTex" OutName="fingers" Format="ci8" Width="16" Height="16" Offset="0x780" />
+        <Texture Name="gAhgShirtTex" OutName="shirt" Format="i8" Width="8" Height="8" Offset="0x880" />
+        <Texture Name="gAhgShoesTex" OutName="shoes" Format="i8" Width="8" Height="8" Offset="0x8C0" />
+        <Texture Name="gAhgBeltBuckleTex" OutName="belt_buckle" Format="ci8" Width="32" Height="16" Offset="0x900" />
+        <Texture Name="gAhgBeltLoopTex" OutName="belt_loop" Format="ci8" Width="16" Height="16" Offset="0xB00" />
+
+        <DList Name="gAhgBeardedHeadDL" Offset="0x2FF0" />
+        <DList Name="gAhgRightHandDL" Offset="0x3400" />
+        <DList Name="gAhgRightForArmDL" Offset="0x3570" />
+        <DList Name="gAhgRightUpperArmDL" Offset="0x36A0" />
+        <DList Name="gAhgLeftHandDL" Offset="0x37D8" />
+        <DList Name="gAhgLeftForearmDL" Offset="0x3948" />
+        <DList Name="gAhgLeftUpperArmDL" Offset="0x3A78" />
+        <DList Name="gAhgLeftChestDL" Offset="0x3BB0" />
+        <DList Name="gAhgRightFootDL" Offset="0x3DD8" />
+        <DList Name="gAhgRightShinDL" Offset="0x3F18" />
+        <DList Name="gAhgRightThighDL" Offset="0x4048" />
+        <DList Name="gAhgLeftFootDL" Offset="0x4180" />
+        <DList Name="gAhgLeftShinDL" Offset="0x42C0" />
+        <DList Name="gAhgLeftThighDL" Offset="0x43F0" />
+        <DList Name="gAhgPelvisDL" Offset="0x4528" />
+
+        <!-- Assets used for alternate heads left over from OOT -->
+        <Texture Name="gAhg2TLUT" OutName="tlut2" Format="rgba16" Width="16" Height="12" Offset="0x46C8" />
+        <Texture Name="gAhgEar2Tex" OutName="ear2" Format="ci8" Width="8" Height="8" Offset="0x4848" TlutOffset="0x46C8" />
```

Comment:
Does this look any differently from `gAhgEarTex`? Or is it used in a different context? I'd prefer to name it based on looks or context than just putting a number there, though that might be the only sensible thing.

Reply:
From my eye they look identical, diff says they are not quite the same
![Screenshot_2023-01-25_16-10-04](https://user-images.githubusercontent.com/42048411/214722598-eb7dfdec-14d7-440d-82b0-8030af458d55.png)
in the object.c the second ear is unused, I can rename to UnusedDuplicate

Reply:
Ah, well, if you have to whip out a hex editor to tell that they look different, then that's close enough. `UnusedDuplicate` is fine even if it's not *exactly* the same.